.github/workflows/ci-standard-checks.yml

```diff
@@ -4,20 +4,23 @@ on:
     branches:
       - main
   pull_request:
-    types: [opened, edited, synchronize, reopened]
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - ready_for_review
     branches:
       - main
-
 jobs:
   ci-standard-checks:
-    runs-on: [ubuntu-latest]
-
+    runs-on:
+      - ubuntu-latest
```

Comment:
Should this run on `ubuntu-latest` or on [self-hosted automated-checks-ephemeral runner](https://github.com/Typeform/share-web/blob/main/.github/workflows/ci-standard-checks.yml#L19C35-L19C35)?

Reply:
good question, it was running on ubuntu-latest previously, and the same for many other repos we have...

Reply:
webflow is on ephemeral , so I will change this one to ephemeral too

Reply:
Just a note - we should not run on `ubuntu-latest` as those are more expensive than our runners. They are free for public repos only.

Reply:
FYI this repo is public, so `ubuntu-latest` makes sense